paasta_tools/kubernetes_tools.py

```diff
@@ -2405,6 +2413,21 @@ def list_deployments(
     ]
 
 
+def list_deployments_in_managed_namespaces(
+    kube_client: KubeClient,
+    label_selector: str,
+) -> List[KubeDeployment]:
+    return [
+        d
+        for namespace in get_all_managed_namespaces(kube_client)
+        for d in list_deployments(
+            kube_client=kube_client,
+            label_selector=label_selector,
+            namespace=namespace,
+        )
```

Comment:
It occurs to me we might end up hitting paasta api 500s again if someone ever accidentally adds a new namespace with `paasta.yelp.com/managed=true` via flux (or otherwise) and forgets the paasta-api-service-per-namespace binding...
Do you think we could still do the `try: .. except 403: pass` option you mentioned previously?